script/install.sh

```diff
@@ -80,7 +82,7 @@ retry() {
     for i in {0..4}; do
         if "$@"; then
             break
-        elif [ $i -lt 4 ]; then
+        elif [ "$i" -lt 4 ]; then
```

Comment:
`i` is a number, not a string.

Reply:
correct.
if I'm nor wrong, I had it like you suggest it now initially but "shellcheck" complained so I made it as it is... also here, feel free to change it to whatever fit's your standards / requirements...